pc/user.cpp

```diff
@@ -204,22 +206,18 @@ void user::parse_upd_price( uint32_t tok, uint32_t itok )
     if ( 0 == (ntok = jp_.find_val( ptok, "status" ) ) ) break;
     symbol_status stype = str_to_symbol_status( jp_.get_str( ntok ) );
 
-    // submit new price
-    if ( sptr->update( price, conf, stype ) ) {
-      // create result
-      add_header();
-      jw_.add_key( "result", 0UL );
-      add_tail( itok );
-    } else if ( !sptr->get_is_ready_publish() ) {
-      add_error( itok, PC_JSON_NOT_READY,
-          "not ready to publish - check rpc / pyth_tx connection" );
-    } else if ( !sptr->has_publisher() ) {
-      add_error( itok, PC_JSON_MISSING_PERMS, "missing publish permission" );
-    } else if ( sptr->get_is_err() ) {
-      add_error( itok, PC_JSON_INVALID_REQUEST, sptr->get_err_msg() );
-    } else {
-      add_error( itok, PC_JSON_INVALID_REQUEST, "unknown error" );
+    // Add the updated price to the pending updates
+    sptr->update_no_send( price, conf, stype, false );
+    if(std::find(pending_vec_.begin(), pending_vec_.end(), sptr) == pending_vec_.end()) {
+      pending_vec_.emplace_back( sptr );
     }
+
+    // Send the result back
+    //  TODO: should we send this result? As the semantics have now changed.
```

Comment:
this seems fine -- I think we don't really have a choice, because not sending this result would change the API.